actor/builtin/paymentbroker/paymentbroker_test.go

```diff
@@ -814,17 +913,16 @@ func (sys *system) retrieveChannel(paymentBroker *actor.Actor) *PaymentChannel {
 	return channel
 }
 
-func (sys *system) applySignatureMessage(target address.Address, amtInt uint64, validAt *types.BlockHeight, nonce uint64, method string, height uint64) (*consensus.ApplicationResult, error) {
+func (sys *system) applySignatureMessage(target address.Address, amtInt uint64, validAt *types.BlockHeight, nonce uint64, method string, height uint64, condition *types.Predicate, suppliedParams ...interface{}) (*consensus.ApplicationResult, error) {
```

Comment:
These helper methods are really hard to follow and poorly documented. Could you leave these in a better state than you found them by documenting the ones touched? Adding more args here has made things slightly less transparent.

Reply:
documented